<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NDEF Message Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QWidget" name="status" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="touchRetrieve">
        <property name="text">
         <string>Retrieve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="touchStore">
        <property name="text">
         <string>Store</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadMessage">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveMessage">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearMessage">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRecord">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>553</width>
         <height>335</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>saveMessage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>touchRetrieve</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>touchReceive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>touchStore</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>touchStore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadMessage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearMessage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveMessage()</slot>
  <slot>loadMessage()</slot>
  <slot>touchReceive()</slot>
  <slot>touchStore()</slot>
  <slot>clearMessage()</slot>
 </slots>
</ui>
